packages/widgetbook/lib/src/fields/iterable_segmented_field.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+
+import 'fields.dart';
+
+/// A [Field] that builds [SegmentedButton]<[Iterable<T>]> for [Iterable] values.
+class IterableSegmentedField<T> extends Field<Iterable<T>> {
+  /// Creates a new instance of [IterableSegmentedField].
+  IterableSegmentedField({
+    required super.name,
+    required this.values,
+    required super.initialValue,
+    this.labelBuilder = defaultLabelBuilder,
+    this.emptySelectionAllowed = true,
+    this.style,
```

Comment:
I have missed this `style` param before. `Field`s should not have any sort of unparasable state. They are designed in a way that makes them unaware of their context.

The fields can be styled using the [`WidgetbookTheme`](https://docs.widgetbook.io/configure/widgetbook-theme), and not individually.

Let's remove it for now, as it's also not present on any other field.
If you find any trouble with the theming alternative, we can discuss it in separate issue/PR.